migration/sqls/000009_add_scheduled_message.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS `scheduled_message` (
+    `id` VARCHAR(255) NOT NULL PRIMARY KEY,
+    `messageId` VARCHAR(255) NOT NULL,
+    `payload` MEDIUMTEXT NOT NULL,
+    `contentType` VARCHAR(255) NOT NULL,
+    `priority` INTEGER NOT NULL,
+    `channelId` VARCHAR(255) NOT NULL,
+    `producerId` VARCHAR(255) NOT NULL,
+    `headers` JSON NULL,
+    `dispatchSchedule` DATETIME NOT NULL,
+    `dispatchedDate` DATETIME NULL,
```

Comment:
Should we consider naming the field `dispatchedAt`? (as it's a datetime value, and we used `createdAt`, `updatedAt` for similar datetime fields)